main.sql

```diff
@@ -242,35 +247,41 @@ CREATE TABLE mag.Papers(
     Rank INT,
     Doi VARCHAR(255),
     DocType VARCHAR(20),
+    Genre TEXT,
+    IsParatext BOOLEAN,
     PaperTitle TEXT,
     OriginalTitle TEXT,
     BookTitle TEXT,
     Year SMALLINT,
-    Date DATE,
-    OnlineDate DATE,
+    Date TEXT,
```

Comment:
Several *Date columns are labeled "t" in the OpenAlex schema and "DateTime" in the MAG schema. I don't know if this is an intentional change - some of the text values that show up sort of make sense as a reason there wouldn't be a date, like "Retracted Publication". Others not so much. 

Any I changed to TEXT here needed to be changed.